go/cmd/ct-fetch/ct-fetch.go

```diff
@@ -393,7 +393,7 @@ func (ld *LogSyncEngine) insertCTWorker() {
 			precert = true
 		}
 
-		if err != nil {
```

Comment:
Should we still log `err` as a warning if it's non-nil, maybe?

Reply:
sure!